### ISSUE TITLE
group-include filter enhencement

### ISSUE THREAD

**Issue by nt74** [huxuan/iptv-m3u-filter#29]
Is it possible to insert a group-include filter with a multiple whitelist (regular expressions)?

Either by; 
(1)
Comma separation reg.expression i.e. --group-include "News","Movies","Sports NBA"

(2)
A whitelist file which can contain all group-id.s to include, i.e. 
News
Movies
Sports NBA

**Comment by huxuan** [comment]
Yes, it is possible, but actually I do not think it is necessary as `group include` currently support regex, you can write a somehow complicated regex to match those whitelists. What do you think of it?

**Comment by nt74** [comment]
I tested some regex expressions out and they do work. So my answer will be Yes I am satisfied.
However in the documentation maybe there should be more information on how to use the regex syntax.

For example I used this expression to solve the above example:
--group-include 'News|Movies|\ASports NBA'

Should there be a special character like i.e. "+", this can be excluded (or included) in the regex like:
--group-include 'News|Movies|\ASports NBA|\ASports[+] NHL'

**Comment by huxuan** [comment]
Thanks for the response.
I will take a look and keep this issue open until it is properly handled.
